coralproject/cay
# clear search filter state when returning to the Create Search screen.…

## What does this PR do?

fixes https://github.com/coralproject/trust/issues/2

Added an action that should clear the filter state upon navigation back to the Create Search or Saved Search screens
## How do I test this PR?
- go to the Create Search screen
- change some filters
- go to another page
- go back to the Create Search screen
- the filters should be defaults

@coralproject/frontend

… linting
